index.js

```diff
@@ -5,35 +5,12 @@
   /* eslint comma-dangle: ["off"], no-var: ["off"], strict: ["error", "function"] */
   /* global self */
 
-  var mapping = {
-    equals: 'fantasy-land/equals',
-    lte: 'fantasy-land/lte',
-    compose: 'fantasy-land/compose',
-    id: 'fantasy-land/id',
-    concat: 'fantasy-land/concat',
-    empty: 'fantasy-land/empty',
-    invert: 'fantasy-land/invert',
-    filter: 'fantasy-land/filter',
-    map: 'fantasy-land/map',
-    contramap: 'fantasy-land/contramap',
-    ap: 'fantasy-land/ap',
-    of: 'fantasy-land/of',
-    alt: 'fantasy-land/alt',
-    zero: 'fantasy-land/zero',
-    reduce: 'fantasy-land/reduce',
-    traverse: 'fantasy-land/traverse',
-    chain: 'fantasy-land/chain',
-    chainRec: 'fantasy-land/chainRec',
-    extend: 'fantasy-land/extend',
-    extract: 'fantasy-land/extract',
-    bimap: 'fantasy-land/bimap',
-    promap: 'fantasy-land/promap'
-  };
+  var empty = {};
```

Comment:
Shall we remove __index.js__ too?

Reply:
It's fine. [xyz][1] only provides a readme and a shell script (no JavaScript files). npm doesn't mind. :)


[1]: https://github.com/davidchambers/xyz

Reply:
It's gone :)